var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -710,6 +716,21 @@ def post_install(self):
         with working_dir(self.build_directory):
             install_tree("bin", join_path(self.prefix, "libexec", "llvm"))
 
+    @classmethod
```

Comment:
I don't think this should be a class method and we shouldn't use `which` because `self.prefix.bin` isn't guaranteed to be in the `PATH`. We should instead use `lc = Executable(self.prefix.bin.join('llvm-config'))`

Reply:
That’s a good point, I’ll rework it.